pkgs/by-name/ca/castero/package.nix

```diff
@@ -39,7 +39,11 @@ python3.pkgs.buildPythonApplication rec {
     "tests"
   ];
 
-  disabledTests = [ "test_datafile_download" ];
+  disabledTests = [
```

Comment:
When possible, put comments describing why the tests were disabled.

Optionally, out of courtesy, inform upstream about the errors.